lib/puppetserver/ca/action/import.rb

```diff
@@ -65,40 +79,29 @@ def run(input)
           return 0
         end
 
-        def log_possible_errors(maybe_errors)
-          errors = Array(maybe_errors).compact
-          unless errors.empty?
-            @logger.err "Error:"
-            errors.each do |message|
-              @logger.err "    #{message}"
-            end
-            return true
+        def check_flag_usage(results)
+          if results['cert-bundle'].nil? || results['private-key'].nil? || results['crl-chain'].nil?
+            '    Missing required argument' + "\n" +
+            '    --cert-bundle, --private-key, --crl-chain are required'
           end
         end
 
-      def check_flag_usage(results)
-        if results['cert-bundle'].nil? || results['private-key'].nil? || results['crl-chain'].nil?
-          '    Missing required argument' + "\n" +
-          '    --cert-bundle, --private-key, --crl-chain are required'
-        end
-      end
-
-      def parse(args)
-        results = {}
-        parser = self.class.parser(results)
+        def parse(args)
+          results = {}
+          parser = self.class.parser(results)
 
-        errors = CliParsing.parse_with_errors(parser, args)
+          errors = CliParsing.parse_with_errors(parser, args)
 
-        if check_flag_usage(results)
-          errors << check_flag_usage(results)
-        end
+          if check_flag_usage(results)
```

Comment:
I think elsewhere we updated this to be
```rb
if error = check_flag_usage(results)
  errrors << error
end
```
if we did that we probably want to do the same here for consistency, though I could see us preferring
```rb
error = check_flag_usage(results)
errors << error if error
```

Reply:
I like the explicit block better, I kind of don't like the post-condition syntax most of the time. I'll update this to be like it is elsewhere.